content/docs/1.10.0/nodes-and-volumes/volumes/create-volumes.md

```diff
@@ -123,25 +123,29 @@ By default the StorageClass for Longhorn created PV/PVC is `longhorn-static`. Us
 
 Users need to manually delete PVC and PV created by Longhorn.
 
-
 ### PV/PVC Creation for Existing Longhorn Volume
 
 Now users can create PV/PVC via our Longhorn UI for the existing Longhorn volumes.
 Only detached volume can be used by a newly created pod.
 
 ### The Failure of the Longhorn Volume Creation
 
-Creating a Longhorn volume will fail if there are no available nodes, disks, or insufficient storage. The failures are categorized into:
-- insufficient storage,
-- disk not found,
-- disks are unavailable,
-- failed to retrieve scheduling settings failed to retrieve,
-- tags not fulfilled,
-- node not found,
-- nodes are unavailable,
-- none of the node candidates contains a ready engine image,
-- hard affinity cannot be satisfied,
-- replica scheduling failed.
+Creating a Longhorn volume can fail for different reasons. The issues are categorized into:
+
+- insufficient storage
+- disk not found
+- disks are unavailable
+- failed to retrieve scheduling settings failed to retrieve
+- tags not fulfilled
+- node not found
+- nodes are unavailable
+- none of the node candidates contains a ready engine image
+- hard affinity cannot be satisfied
+- replica scheduling failed
+- unused failed replica is not supported
+- replica already scheduled
+- data store operation failed
```

Comment:
Assume this case is related to restoring a backup volume.

Reply:
Ah, it is Longhorn datastore operations (listNodes, getNode...)

Reply:
Renaming to `longhorn client operation failed` to make it clear.